src/XMakeTasks/Microsoft.Xaml.targets

```diff
@@ -13,7 +13,7 @@
         the .NET Framework directory and thus will no longer be the right answer. Override it to point
         to the correct .NET Framework location. --> 
    <PropertyGroup>
-      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildFrameworkToolsPath)</XamlBuildTaskPath>
+      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildToolsPath)</XamlBuildTaskPath>
```

Comment:
The inclusion of the framework version of Microsoft.Xaml.targets is done on the import below this property group. The XamlBuildTaskPath is used as the ApplicationBase for AppDomains. Setting XamlBuildTaskPath to the framework directory causes the new XamlBuildTask AppDomains to attempt to load assemblies from the framework directory.